service/test/controllers/HealthControllerMockTest.java

```diff
@@ -0,0 +1,34 @@
+package controllers;
+
+import static org.junit.Assert.assertEquals;
+
+import akka.actor.ActorRef;
+import javax.ws.rs.core.Response;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PowerMockIgnore;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+import play.mvc.Http;
+import play.mvc.Result;
+import scala.concurrent.Await;
+
+@RunWith(PowerMockRunner.class)
```

Comment:
Please be aware - when you run with mock, we cannot get coverage. It helps just to test out how the logic works.